flake.crane.nix

```diff
@@ -471,6 +471,7 @@ rec {
       "crypto/tbs"
       "crypto/hkdf"
       "fedimint-bitcoind"
+      "fedimint-bin-tests"
```

Comment:
I don't this is a good idea?

Reply:
Why? Doesn't seem like a problem.

Reply:
because client shouldn't need `fedimint-bin-tests` like it doesn't need `fedimint-testing`.

Reply:
Agreed, but these filters here are entirely for caching purposes (avoiding building stuff if completely unrelated thing changed). If there's an unwanted dependency somewhere, this is the least of the problems.